src/lib/telemetry/TelemetryCollector.ts

```diff
@@ -1,10 +1,15 @@
+import { randomBytes } from 'crypto'
+import { writeJson, ensureFile } from 'fs-extra'
 import { spawn } from 'child_process'
 import { join } from 'path'
+
 import * as pkgJson from '../../../package.json'
 import { ErrorReport } from '../error/ErrorReport'
 import { ITelemetryLocalStore, TelemetryLocalStore } from './TelemetryStore'
 
 export class TelemetryCollector {
+  private static readonly REMOTE_FLUSH_INTERVAL = 1000 * 60 * 10 // Ten minutes
+  private static readonly telemetryObjFilePathPrefix = '~/.config/configstore/vtex/telemetry'
```

Comment:
Since we are going to use this directory to everything related to telemetry, how about we change the name to something like `LOCAL_TELEMETRY_DIR` or something better?

The `conf.ts` file exports a `configDir`, so this could be:
```ts
private static readonly LOCAL_TELEMETRY_DIR = join(configDir, 'vtex', 'telemetry')
```

Reply:
Just saw that actually we are not using this dir to everything, how about we use? What do you think?
```
vtex/
  +-- telemetry/
    +-- errors/
    +-- hash1.json
    +-- hash2.json
```